elements/lisk-p2p/src/utils/sanitize.ts

```diff
@@ -47,6 +51,20 @@ export const sanitizeInitialPeerInfo = (peerInfo: ProtocolPeerInfo) => ({
 	wsPort: peerInfo.wsPort,
 });
 
+export const sanitizeInternalPeerInfo = (
```

Comment:
`sanitizeInternalPeerInfo` -> `sanitizeEnhancedPeerInfo` or the other way around. Right now, I don't clearly understand the difference in between `internalState` and `enhancedInfo`. Shall we merge them?